APIScrape/scrape.php

```diff
@@ -112,7 +113,8 @@ function getCatImageURLs(array $catBreeds):array
         ));
         $imageApiResponse = curl_exec($curl);
         if($imageApiResponse === false) {
-            echo 'Curl error: ' . curl_error($curl);
+            $imageApiResponseError = 'Curl error: ' . curl_error($curl);
+            exit($imageApiResponseError);
```

Comment:
don't we need to exit the script itself rather than exiting response error?

Reply:
Not how exit works